lib/reporter.js

```diff
@@ -212,21 +212,21 @@ var generateReport = function (options) {
                     if (step.embeddings !== undefined) {
                         step.embeddings.forEach(function (embedding) {
 
-                            if (embedding.media.type === 'text/plain') {
+                            if (embedding.mime_type === 'text/plain') {
```

Comment:
How about adding backward-compatibility with, and use `embeddingType` for if-else?

```
var embeddingType = {};

if (embedding.mime_type) {
   embeddingType = embedding.mime_type.type;
} else if (embedding.media) {
   embeddingType = embedding.media.type;
}
```